dnf.spec

```diff
@@ -261,6 +263,14 @@ pushd build-py3
 popd
 %endif
 
+%pre yum
```

Comment:
honestly this script doesn't conform any guidelines... https://fedoraproject.org/wiki/Packaging:Directory_Replacement

Apparently there is a warning:

> Never use this for directories where users or other packages might add or modify files.
> e.g. in subdirectories of /etc or /var or in directories where other software is expected to drop in plugins, configuration, or data.

So I'm not sure what correct solution would be.


Also you probably wanted yum/yum4 conditional because on rhel<=7 this package is named yum4

Reply:
also it should not be %pre, but %pretrans